github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -2183,7 +2183,7 @@ presubmits:
         - automation/test.sh
         env:
         - name: TARGET
-          value: k8s-1.26-sig-network
+          value: k8s-1.26-centos9-sig-network
```

Comment:
```suggestion
          value: k8s-1.26-centos9-sig-network-no-istio
```

This way we could drop https://github.com/kubevirt/project-infra/pull/2592 and save ourselves from rebasing unnecessarily.

Reply:
(I'm ok with handling this in a different PR too, don't feel obliged to block this PR on my review)

Reply:
Updated, PTAL, thank you for pointing this out, makes total sense!

Reply:
Thanks o/